### DISCUSSION TITLE
[Question][Hw3] Can we 1) use stdlib predefined type, and 2) can define own module?

### DISCUSSION THREAD

**Discussion by Re-st** [prosyslab-classroom/cs348-information-security#698] [Q&A]
Name: Geon Park

1)
In lesson0 ppt, it says 'stdlib module is automatically opened'.
And it is.
Can we use stdlib predefined type (eg. Float) & functions (eg. truncate) ?

2)
Homework2 was easy to handle functions since we were able to use modules - https://github.com/prosyslab-classroom/cs348-information-security/issues/246
But we also shouldn't "change directory structures " as described on each hw's readme.
Can we define own module in the code, without hurting (if) existing modules? (Hurting would include changing existing modules' methods' any detail like input / output type, etc.)

**Comment by yeonhee-ryou** [comment]
1. You can use anything as long as it can be compiled in a given environment. Check the execution results of Gradescope.
2. You can define your own module inside the given file. Do not add or rename the given files and directories.

**Comment by Re-st** [comment]
Thank you!
